### DISCUSSION TITLE
how to disable proxy auto forward pop up

### DISCUSSION THREAD

**Discussion by Oliver-Weng-Sorcerer** [coder/code-server#5802] [General]
Hi @code-asher 

do you know by any chance how to disable the pop up on the lower right corner when web is served under a particular port ?

Or disable the whole "Ports" feature completely ?

Thanks

<img width="1270" alt="Screenshot 2022-11-21 at 11 27 43 AM" src="https://user-images.githubusercontent.com/28814939/203107942-5970f0d1-75fb-4134-bd70-9ff9424ac764.png">

<img width="954" alt="Screenshot 2022-11-21 at 11 28 06 AM" src="https://user-images.githubusercontent.com/28814939/203107938-2a7fb1bc-f202-4ce4-b594-7061aecaeb4b.png">

**Comment by jsjoeio** [comment]
I don't know if there is a way to do it currently but we could add a CLI flag. Do you want to open an issue about that?

**Reply by Oliver-Weng-Sorcerer** [reply]
will do
